demos/GMW_demo/gmw_open.js

```diff
@@ -0,0 +1,132 @@
+/**
+ * the function to reconstruct the result from the secret.
+ * @ignore
+ * @function gmw_reconstruct
+ * @param {module:jiff-client~JIFFClient} jiff - the jiff instance
+ * @param {number} secret - the secretshare to be open.
+ * @returns {object} the opened result
+ */
+function gmw_reconstruct(jiff,shares) {
+  var ls=[];
+  for (let [key] of Object.keys(shares)) {
+    ls.push(shares[key]['value']);
+
+  }
+  var re=ls[0];
+  for (var i=1;i<Object.keys(ls).length;i++) {
+    re=re^ls[i];
+  }
+  return re;
+}
+/**
+ * Share the given share to all the parties in the jiff instance.
+ * @function jiff_broadcast
+ * @ignore
+ * @param {module:jiff-client~JIFFClient} jiff - the jiff client instance.
+ * @param {module:jiff-client~JIFFClient#SecretShare} share - the share.
+ * @param {Array} parties - the parties to broadcast the share to.
+ * @param {number|string} op_id - a unique operation id, used to tag outgoing messages.
+ *
+ */
+var jiff_broadcast = function (jiff, share, parties, op_id) {
```

Comment:
as we move into the library we can remove this duplicate function and re-use the already existing jiff_broadcast.

Reply:
I'm not sure where is the jiff_broadcast function internal interface in the lib which can be directly called and used by gmw_open here.